pages/dkp/kommander/2.2/projects/applications/platform-applications/index.md

```diff
@@ -1,18 +1,14 @@
 ---
 layout: layout.pug
-navigationTitle: Platform Applications
-title: Platform Applications
+navigationTitle: Project Platform Applications
+title: Project Platform Applications
 menuWeight: 10
-excerpt: How Platform applications work
+excerpt: How project Platform applications work
```

Comment:
```suggestion
excerpt: How project platform applications work
```

Reply:
hit the merge too soon before seeing these 🤦🏻‍♀️